src/main.ts

```diff
@@ -227,4 +229,53 @@ function defaultLabelCheck(){
       core.setFailed('Please set default label as need-to-triage')
   })
 }
+
+async function standardLabelsCheck(repository: string, ownername: string, secret_token: string, octokit: Octokit){
+  try{
+    const result = await octokit.request('GET /repos/{owner}/{repo}/labels',{
+      repo: repository,
+      owner: ownername,
+      headers : { 
+        Authorization: 'Bearer ' + secret_token,
+      },
+    }); 
+    let labelArray = result.data;
+    var map = new Map();
+    var absentLabels = new Array();
+    for(let i=0; i<labelArray.length; i++){
+      map.set(labelArray[i].name , 1);
+    }
+    standardLabelsCheckHelper('need-to-triage', map, absentLabels);
```

Comment:
Iterate over the values in a loop

Reply:
Should I use nested loops instead of map?

Reply:
Fixed!